includes/discovery/os/firebox.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+if (starts_with($sysDescr, 'WatchGuard Fireware') || starts_with($sysObjectId, '.1.3.6.1.4.1.3097.1.5')) {
```

Comment:
There's no watchguard definition so that shouldn't matter now I've removed the disco file.

I couldn't say about firebox / fireware as I don't know what those are but happy to merge them into one.